libs/sdk-ui-ext/styles/internal/scss/dashboard_embedding.scss

```diff
@@ -184,9 +184,13 @@
     position: relative;
     z-index: 1;
     height: ($item-headline-lineHeight + $item-headline-padding-vertical) * 2 + 2; // relate SD-956: please update HEADLINE_OUTER_HEIGHT in widgetHeightUtil.ts when height is changed
-    margin-top: 10px;
     line-height: ($item-headline-lineHeight + $item-headline-padding-vertical) * 2;
 
+    &.layout-height-small {
+        height: 38px;
```

Comment:
What about to keep it in components code directly to minimize the css involvement?